mlflow/server/js/src/experiment-tracking/components/CompareRunArtifactView.css

```diff
@@ -0,0 +1,17 @@
+.artifact-container {
+  display: flex;
+  flex-direction: row;
+  height: 100vh;
+}
+
+.artifact-right {
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+}
+
+.artifact-right > div {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
```

Comment:
i think we can consolidate all these styles into the `css` property in the react code itself! see [1] in the comments below